### ISSUE TITLE
react with webpack

### ISSUE THREAD

**Issue by drduan** [patr0nus/DeskGap#17]
how react with webpack

**Comment by p32929** [comment]
Yeah, it would've been great to be able to write in react and webpack so that every time file changes, the ui gets refreshed
